src/Stream.ts

```diff
@@ -47,26 +47,30 @@ export default class Stream<T> {
       if (!env.message) {
         return
       }
-      const msg = await decoder(env)
-      // decoder can return undefined to signal a message to ignore/skip.
-      if (!msg) {
-        return
-      }
-      // Check to see if we should update the stream's content topic subscription
-      if (contentTopicUpdater) {
-        const topics = contentTopicUpdater(msg)
-        if (topics) {
-          this.resubscribeToTopics(topics)
+      try {
```

Comment:
Needed to wrap this in a try/catch since otherwise the errors would bubble up if an invalid message was found in the stream.